NICE.Identity.Management/ClientApp/src/components/FilterBox/FilterBox.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+
+import {
+	FilterPanel,
+	FilterGroup,
+	FilterOption,
+} from "@nice-digital/nds-filters";
+
+import "@nice-digital/nds-filters/scss/filters.scss";
+import { capitaliseWord } from "../../helpers/capitaliseWord";
+
+type FiltersType = {
+	name: string;
+	value: string;
+};
+
+type FilterBoxProps = {
+	name: string;
+	filters: Array<FiltersType> | Array<string>;
+	selected: Array<string>;
+	onCheckboxChange: (e: string) => void;
+};
+
+export const FilterBox = (props: FilterBoxProps): React.ReactElement => {
+	let filters: any = [];
+	let selectedCount = 0;
+
+	if (typeof props.filters[0] === "string") {
+		props.filters.map((filter) => {
+			filters.push({
+				value: filter,
+				name: capitaliseWord(filter as string),
```

Comment:
capitaliseWord... why? is this just the word the user's typed in? why capitalise?